code/modules/antagonists/heretic/knowledge/sacrifice_knowledge/sacrifice_knowledge.dm

```diff
@@ -151,6 +151,15 @@
 			valid_targets -= department_mind
 			break
 
+///Monke Edit begin - Heretics now always get another heretic as a sacrifice target (if there is one)
+	for(var/datum/mind/heretic_mind as anything in shuffle(valid_targets))
+		var/datum/antagonist/heretic/heretic_sac = heretic_mind.has_antag_datum(/datum/antagonist/heretic)
+		if(heretic_sac)
```

Comment:
```suggestion
		if(IS_HERETIC(heretic_mind))
```

Reply:
This doesn't work because its picking from minds not mobs for your target

Reply:
Oh, fair. I misread the code.